### DISCUSSION TITLE
Issue with MDF3 when using the option remove_source_from_channel_names and channel selection

### DISCUSSION THREAD

**Discussion by ZdP13** [danielhrisca/asammdf#721] [Q&A]
Hello, I use asammdf mostly with MDF4 files. Sometimes some colleagues come with a MDF3 so I try to keep my scripts compatible with it. They have the issue that the source name is included in the channel name, so I use the (very helpful) remove_source_from_channel_names argument when reading them. However, I cannot use it in combination with channel selection (with the channels argument) as it then still expects channel names with the source included. Is there an easy solution ? Thanks!

**Comment by danielhrisca** [comment]
@ZdP13  please try the development branch code

**Reply by ZdP13** [reply]
Hi @danielhrisca. Thanks for the quick feedback. After several trial, I wasn't able to install the development version in the protected environment of my company -> I will wait for the next release
